### PR TITLE
Add ClonesPlus extension

### PR DESCRIPTION
**Pull request by LukeManiaStudioss**
ClonesPlus adds various blocks to make the cloning features on Scratch far more useful.

Extension thumbnail (image and uncropped SVG):
image

ClonesPlus

### CODE REVIEW

File: website/index.ejs [hunk 1]
```diff
@@ -441,9 +429,15 @@
         </div>
 
         <div class="extension">
-          <%- banner('utilities') %>
-          <h3>Utilities</h3>
-          <p>A bunch of interesting blocks. Originally created by <a href="https://scratch.mit.edu/users/Sheep_maker/">Sheep_maker</a>.</p>
+          <%- banner('clonesplus') %>
```
Comment (GarboMuffin) [inline website/index.ejs:432]: banner('LukeManiaStudios/ClonesPlus')

Comment (LukeManiaStudioss) [reply website/index.ejs:432]: oops 💀

Comment (LukeManiaStudioss) [reply website/index.ejs:432]: I'll get used to this eventually 😭

Comment (GarboMuffin) [reply website/index.ejs]: eventually we will clean up the mess that is index.ejs. a mistake like this really should not be able to have "All checks have passed"

File: website/index.ejs
Comment (GarboMuffin) [inline website/index.ejs:1]: your formatter made more changes to this file than it should've

Comment (LukeManiaStudioss) [reply website/index.ejs:1]: You think? 😭

**Comment by LukeManiaStudioss** [comment]
Are there any more issues with this that I should address before it gets merged?

**Comment by GarboMuffin** [comment]
ended up making quite a few changes to prevent renaming variables from causing as many issues (menus now use variable IDs instead of names); consider taking a look to see if there's any regressions

**Comment by LukeManiaStudioss** [comment]
First of all, I had not even considered renaming variables so I appreciate you not only pointing that out but resolving it yourself, that was impeccably helpful

I just tried it, and I have no problems with it. Try as I might I can't break it lol

**Comment by David-Orangemoon** [comment]
LETS GOOOOOOO.

**Comment by UndermanReal** [comment]
I think you should also add a "touching clone with (var) containing (value)?"

**Comment by LukeManiaStudioss** [comment]
1. Make this an issue
2. That block already exists, with limited functionality

**Pull request merged**